### PR TITLE
simpler allSettled implementation

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: client/components/onward-journey/main.js [hunk 1]
```diff
@@ -85,7 +75,9 @@ module.exports.init = () => {
 		);
 	}
 
-	return allSettled(fetchPromises)
+	return Promise.all(
+		fetchPromises.map(p => p.catch(() => null))
+	)
 		.then(() => {
 			lazyLoadImages();
```
Comment (andygnewman) [inline client/components/onward-journey/main.js]: Given this is client side I'm not clear why we're adding lazy loading on as well. Will do another PR to clean that up.

**Comment by andygnewman** [comment]
LGTM

File: client/components/onward-journey/main.js (hunk 1 above)
Comment (wheresrhys) [inline client/components/onward-journey/main.js]: that's funny as the original intent of this PR was to remove that (replacing with something from n-ui) but it _is_ needed

Comment (ironsidevsquincy) [inline client/components/onward-journey/main.js]: yeah, to lazy load the images in the more ons
